firefox-ios/Client/Frontend/Browser/WebView/WebviewViewController.swift

```diff
@@ -5,13 +5,35 @@
 import Foundation
 import Shared
 import WebKit
+import Common
 
-class WebviewViewController: UIViewController, ContentContainable, ScreenshotableView {
+class WebviewViewController: UIViewController,
+                             ContentContainable,
+                             ScreenshotableView,
+                             Themeable,
+                             InjectedThemeUUIDIdentifiable {
+    private struct UX {
+        static let documentLoadingViewAnimationDuration: CGFloat = 0.3
+    }
+    private var documentLoadingView: TemporaryDocumentLoadingView?
     private var webView: WKWebView
     var contentType: ContentType = .webview
+    var themeManager: any ThemeManager
+    var notificationCenter: any NotificationProtocol
+    var themeObserver: (any NSObjectProtocol)?
```

Comment:
I don't think the `any` are necessary?
```suggestion
    var themeManager: ThemeManager
    var notificationCenter: NotificationProtocol
    var themeObserver: NSObjectProtocol?
```